stories/module-search-console-components.stories.js

```diff
@@ -66,3 +69,21 @@ generateReportBasedWidgetStories( {
 	options: pageDashboardImpressionsArgs,
 	component: DashboardImpressionsWidget,
 } );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'search-console',
+	datastore: STORE_NAME,
+	group: 'Search Console Module/Components/Dashboard/Popular Keywords Widget',
+	data: dashboardPopularKeyWordsWidgetData,
+	options: dashboardPopularKeyWordsWidgetArgs,
+	component: DashboardPopularKeywordsWidget,
+} );
+
+generateReportBasedWidgetStories( {
+	moduleSlug: 'search-console',
+	datastore: STORE_NAME,
+	group: 'Search Console Module/Components/Page Dashboard/Popular Keywords Widget',
+	data: dashboardPopularKeyWordsWidgetData,
+	options: dashboardPopularKeyWordsWidgetArgs,
+	component: DashboardPopularKeywordsWidget,
+} );
```

Comment:
Based on the above comment, in both of these tests, `wrapWidget: false` needs to be specified too for proper appearance within Storybook.

Reply:
updated